### DISCUSSION TITLE
Redacting documents

### DISCUSSION THREAD

**Discussion by 42Joe** [Stirling-Tools/Stirling-PDF#637] [Q&A]
Is there a way to redact lines e..g. in invoices manually? Auto-redacting does not help for this purpose.

**Comment by Frooodle** [comment]
by lines what do you mean? like "blank out line 2 and 4"
or mannually drawing boxes on areas to black out?
Can you be more descriptive

**Comment by 42Joe** [comment]
What I need is a possibility to blank out lines, e.g. in my credit card invoice, in case I need to attach it to my employer. I want to show only the relevant debits. Only hiding is not enough, the information has to be deleted. This is simple in Acrobat.

**Reply by Frooodle** [reply]
"This is simple in Acrobat."
How does acrobat handle this? select line and it redacts? Or do you draw a box over it etc?
I want to know how you want this implemented

**Reply by 42Joe** [reply]
Hi,
thanks for your quick replies. Well, the handling can be found here:
https://www.adobe.com/acrobat/hub/redact-pdf-on-pc.html
Best, 
Joe

**Comment by josh-stephens** [comment]
Yes, in Acrobat Pro, you click on the Redact tool, and then you get a text selection cursor, and every time you click, drag, and let go, the selected text gets a red box drawn around it. You can move throughout the document selecting all of the text that you want, then you click the Redact button, and all of the boxes go from red outlines to solid black, and a Save dialog box pops up with "filenameRedacted.pdf" highlighted. This is a really convenient workflow, and the only thing I've found missing from the otherwise stellar StirlingPDF toolkit.

**Comment by Frooodle** [comment]
Good news! We have a brand new redact tool coming out in next release! its already merged and ready to go,
